src/ext/peek-source.lisp

```diff
@@ -43,7 +43,12 @@
 
 (define-key *peek-source-keymap* "Return" 'peek-source-select)
 (define-key *peek-source-keymap* 'next-line 'peek-source-next)
+(define-key *peek-source-keymap* "C-n" 'peek-source-next)
 (define-key *peek-source-keymap* 'previous-line 'peek-source-previous)
+(define-key *peek-source-keymap* "C-p" 'peek-source-previous)
```

Comment:
`C-n/C-p`
Even without key bindings, I think the cursor position is displayed automatically.
Is the current behavior not enough?

Reply:
The current behavior works fine with the up and down keys.
However we don't have, by default, a movement key on the home row, that's why I was looking for more movement keys.

I realize that C-n and C-p are not appropriate, because we can edit the buffer, so they will overwrite the editor keys (same issue with "q" and "?").

I will try later to make these keys work only on the read-only text,

for now I remove them.